monitor_correct_errors/models/monitor_correct_errors.py

```diff
@@ -0,0 +1,336 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields
+from odoo.addons.child_compassion.models.compassion_hold import HoldType
+
+
+class MonitorCorrectErrors(models.AbstractModel):
+    _name = 'monitor.correct.errors'
+
+    def process_errors(self):
+
+        self.error_log_list = []
```

Comment:
Instead of having an error list, add a field in the model :
Change the class type to models.TransientModel and add the field
`error_ids = fields.Many2many(...)`